corehq/apps/export/models/new.py

```diff
@@ -1730,6 +1823,25 @@ def _add_to_group_schema(group_schema, path_start, prop, app_id, app_version):
         return schema
 
 
+class SMSExportDataSchema(ExportDataSchema):
+    @property
+    def type(self):
+        return SMS_EXPORT
+
+    @classmethod
+    def generate_schema_from_builds(cls, domain, app_id, identifier, force_rebuild=False,
+            only_process_current_builds=False, task=None):
+        return cls(domain=domain)
+
+    @classmethod
+    def schema_version(cls):
+        return SMS_DATA_SCHEMA_VERSION
+
+    @staticmethod
+    def get_latest_export_schema(domain, app_id, case_type):
+        return SMSExportDataSchema(domain=domain)
```

Comment:
since you don't actually return the saved schema, this will create a new schema on every request to an export